src/server/journal/serializer.cc

```diff
@@ -76,7 +76,7 @@ void JournalWriter::Write(const journal::Entry& entry) {
     case journal::Op::SELECT:
       return Write(entry.dbid);
     case journal::Op::PING:
-      return;
+      return Write(entry.lsn);
```

Comment:
you should check here the version of the replica.
we send PING today in replica takeover
If we will send the lsn value to a replica with version < ver4 this will be a problem

Reply:
How does this workwith ver3